src/components/PatientProfilePage/PatientProfilePage.tsx

```diff
@@ -0,0 +1,448 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory, NavLink } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { ValidatorForm } from 'react-material-ui-form-validator';
+import { useMutation } from '@apollo/react-hooks';
+import { useQuery } from 'react-apollo';
+import { useSnackbar } from 'notistack';
+import ConfirmModal from '../common/ConfirmModal';
+import { Colours } from '../../styles/Constants';
+import FormField from '../common/FormField';
+import CompletePatientButton from './CompletePatientButton';
+import RadioSelector from '../common/RadioSelector';
+import TriagePills from './TriagePills';
+import StatusPills from './StatusPills';
+import PatientTransportPage from './PatientTransportPage';
+import {
+  TriageLevel,
+  Status,
+  Patient,
+  Gender,
+  GET_PATIENT_BY_ID,
+  GET_ALL_PATIENTS,
+} from '../../graphql/queries/patients';
+import { ADD_PATIENT, EDIT_PATIENT } from '../../graphql/mutations/patients';
+import { Hospital, GET_ALL_HOSPITALS } from '../../graphql/queries/hospitals';
+import {
+  Ambulance,
+  GET_ALL_AMBULANCES,
+} from '../../graphql/queries/ambulances';
+import { CCP, GET_CPP_BY_ID } from '../../graphql/queries/ccps';
+
+interface FormFields {
+  barcodeValue: string;
+  triage: TriageLevel | null;
+  gender: Gender;
+  age: number | null;
+  notes: string;
+  runNumber?: number | null;
+  collectionPointId?: number;
+  status: Status | null;
+  triageCategory?: number | null;
+  TriageLevel?: number | null;
+}
+
+const PatientProfilePage = ({
+  match: {
+    params: { eventId, ccpId, patientId, barcodeValue },
+  },
+  mode,
+}: {
+  match: {
+    params: {
+      eventId: string;
+      patientId?: string;
+      ccpId: string;
+      barcodeValue?: string;
+    };
+  };
+  mode: string;
+}) => {
+  const history = useHistory();
+  const { enqueueSnackbar } = useSnackbar();
+  const [openTransportModal, setOpenTransportModal] = useState(false);
+  const [openTransportPage, setOpenTransportPage] = useState(false);
+  const [transportConfirmed, setTransportConfirmed] = useState(false);
+  const [transportingPatient, setTransportingPatient] = useState(false);
+  const [selectedHospital, setSelectedHospital] = useState<Hospital>({
+    id: '',
+    name: '',
+  });
+  const [selectedAmbulance, setSelectedAmbulance] = useState<Ambulance>({
+    id: '',
+    vehicleNumber: 0,
+  });
+
+  const { data, loading } = useQuery(
+    mode === 'edit' && patientId
+      ? GET_PATIENT_BY_ID(patientId)
+      : GET_ALL_PATIENTS
+  );
+  const patients: Array<Patient> = data ? data.patient : [];
```

Comment:
```suggestion
  const patients: Array<Patient> = data ? data.patients : [];
```
Fixes issue discussed [here](https://github.com/uwblueprint/paramedics-react/pull/27/files#r481334417)

Reply:
Actually this only fixes the cache not updating... looking into why it's not appearing correctly in CCP dashboard.